website/src/components/expandable/styles.module.css

```diff
@@ -1,6 +1,6 @@
 :local(.link) :local(.headerText) {
     content: '';
-    color: black; /* Black text in normal mode */
+    color: rgba(18, 12, 12, 0.862); /* Black text in normal mode */
```

Comment:
ok the black and white font color were looking too strong for me so i tweaked it a little. this is how it was looking:

![Screenshot 2024-02-16 at 14 54 47](https://github.com/dbt-labs/docs.getdbt.com/assets/89008547/f65bdc4e-5543-44a7-b350-406600818bca)
![Screenshot 2024-02-16 at 14 54 54](https://github.com/dbt-labs/docs.getdbt.com/assets/89008547/ba410522-5034-484d-8417-1efefeae8e87)

Reply:
i updated it so it's similar to the sidebar color.